crawl-ref/source/mon-gear.cc

```diff
@@ -925,6 +926,16 @@ int make_mons_weapon(monster_type type, int level, bool melee_only)
               { SPWPN_DISTORTION,      2 },
               { SPWPN_CHAOS,           1 }, },
         } },
+        { MONS_VAUD, {
+            { { WPN_FLAIL,          35},    //Total weight 100
+              { WPN_MORNINGSTAR,    24},
+              { WPN_LONG_SWORD,     20},
+              { WPN_SCIMITAR,       20},
+              { WPN_TRISHULA,       1},},
+            { 1, 1, 2 },
+            { { SPWPN_HOLY_WRATH, 9 },      //Total weight 10
+              { SPWPN_FLAMING, 1 } },
```

Comment:
Have you tried adjusting the weights as a test to see if this can end up creating a flaming trishula? (It probably shouldn't be able to.)